src/index.html

```diff
@@ -0,0 +1,127 @@
+<html lang="pl">
```

Comment:
```suggestion
<html lang="en">
```

in the page, all texts are in English language, so lang must be set on `en`